bHY6yh-d36617/cod-bxxqqq_bHY6yh-d36617.xml

```diff
@@ -0,0 +1,413 @@
+              <lb ed="#B"/>annis ¶ vir bono et sapiens dig<lb ed="#B" break="no"/>nis 
+              ait esse parendum ¶ Nec tantum 
+              <lb ed="#B"/>ignorat quid distent erua lupinis</quote>
+            </p>
+            <p xml:id="bHY6yh-d36617-d1e301">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d36617-Rd1e335">Ovidius sine titulo seu amorum libro I 
+              <lb ed="#B"/>capitulo 8</ref> <quote xml:id="bHY6yh-d36617-Qd1e305">Crede mihi res est ingentio<lb ed="#B" break="no"/>sa 
+              dare</quote>
+            </p>
+            <p xml:id="bHY6yh-d36617-d1e311">
+              ¶ <ref xml:id="bHY6yh-d36617-Rd1e345">Idem de arte amandi libro 
+              <lb ed="#B"/>3 et finale</ref> <quote xml:id="bHY6yh-d36617-Qd1e313">Munera dat diues</quote>
+            </p>
+            <p xml:id="bHY6yh-d36617-d1e326">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d36617-Rd1e358">MAtheus in ewangelio capitulo 25</ref> <quote xml:id="bHY6yh-d36617-Qd1e330">habenti 
+              <lb ed="#B"/>dabitur et habundat ei autem 
+              <lb ed="#B"/>qui non habet et quod habet aufferetur 
+              <lb ed="#B"/>ab eo</quote>
+            </p>
+            <p xml:id="bHY6yh-d36617-d1e338">
+              ¶ <ref xml:id="bHY6yh-d36617-Rd1e371">Idem dicit lucas in ewangelio capitulo 19</ref>
+            </p>
+            <p xml:id="bHY6yh-d36617-d1e346">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d36617-Rd1e383">Lvcas in ewangelio capitulo 6</ref> <quote xml:id="bHY6yh-d36617-Qd1e350">date et da<lb ed="#B" break="no"/>bitur 
+              uobis</quote>
+            </p>
+            <p xml:id="bHY6yh-d36617-d1e354">
+              ¶ <ref xml:id="bHY6yh-d36617-Rd1e393">8 capitulo</ref> <quote xml:id="bHY6yh-d36617-Qd1e358">Qui habet da<lb ed="#B" break="no"/>bitur 
+              illi et quicumque non habet etiam 
+              <lb ed="#B"/>quod putat se habere aufferetur abeo</quote>
+            </p>
+            <p xml:id="bHY6yh-d36617-d1e364">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d36617-Rd1e405">PAulus apostolus in epistola 2a ad co<lb ed="#B" break="no"/>rinteos 
+              capitulo 9</ref> ¶ <quote xml:id="bHY6yh-d36617-Qd1e368">Illarorem dato<lb ed="#B" break="no"/>rem 
+              diligit deus</quote>
+            </p>
+            <p xml:id="bHY6yh-d36617-d1e374">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d36617-Rd1e416">SEneca de beneficiis<!--line jumps up--> libro I</ref> <quote xml:id="bHY6yh-d36617-Qd1e378">dignus<!--line jumps down--> est decipi 
+              <lb ed="#B"/>qui de recipiendo cogitauit 
+              <lb ed="#B"/>dum daret</quote>
+            </p>
+            <p xml:id="bHY6yh-d36617-d1e385">
+              ¶ <ref xml:id="bHY6yh-d36617-Rd1e428">libro 2</ref> <quote xml:id="bHY6yh-d36617-Qd1e389">dabo egenti sed 
+              <lb ed="#B"/>ut ipse non egeam succurram peri<lb ed="#B" break="no"/>turo 
+              sed ut ipse non peream &
```

Comment:
`&` here; needs to be corrected. Perhaps it is supposed to be a pilcrow? I"m not sure.